samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -77,6 +77,13 @@ tests:
       - nrf9160dk_nrf9160_ns
     platform_allow: nrf9160dk_nrf9160_ns
     tags: ci_build
+  sample.nrf9160.modem_shell.lwm2m_bootstrap:
+    build_only: true
+    extra_args: OVERLAY_CONFIG="overlay-lwm2m.conf;overlay-lwm2m_bootstrap.conf"
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    tags: ci_build
```

Comment:
This is not directly related to this PR but because I didn't see CONFIG_MOSH_LWM2M_PSK set here, I started to check how it's handled. We discussed offline that CONFIG_MOSH_LWM2M_PSK would be set empty by default and BUILD_ASSERT would be made to avoid misconfiguration of an empty value. This means user should always set it when compiling.

Reply:
Pushed a new commit which removes the default key and adds a build assert if the key is not configured.